Expensify/App
# Hardcode credentials on dev if available

I figured this is gonna save us some time while testing 

Test:
- Fill in .env file with PARTNER_USER_ID and PARTNER_USER_SECRET
- Load sign in page
- See credentials populated
![image](https://user-images.githubusercontent.com/165133/91603872-9bc8a800-e922-11ea-8f7a-be3cd92918a3.png)
